agent/eventhandler/task_handler.go

```diff
@@ -58,16 +67,73 @@ type TaskHandler struct {
 	// stateSaver is a statemanager which may be used to save any
 	// changes to a task or container's SentStatus
 	stateSaver statemanager.Saver
+
+	drainEventsFrequency time.Duration
```

Comment:
Looks like this needs gofmt.

Reply:
Why do you think so? It is formatted code.

Reply:
I...have no idea what I was looking at.  Please ignore.